manifests/charts/istio-operator/templates/deployment.yaml

```diff
@@ -12,6 +12,13 @@ spec:
     metadata:
       labels:
         name: istio-operator
+        {{- range $key, $val := .Values.operatorLabels }}
+        {{ $key }}: "{{ $val }}"
+        {{- end }}
+      annotations:
+        {{- if .Values.podAnnotations }}
+{{ toYaml .Values.podAnnotations | indent 8 }}
+        {{- end }}
```

Comment:
Just to avoid empty annotations like [this](https://github.com/istio/istio/pull/34003/files#diff-76349a7ea6a7900cdb39f2f768922855162b2865dfb591f31b78ef5f26405b27R171).